plugin-hrm-form/src/types/types.ts

```diff
@@ -168,6 +168,7 @@ export type Contact = {
   accountSid: string;
   twilioWorkerId: string;
   number: string;
+  caseId?: number | null;
```

Comment:
```suggestion
  caseId?: number;
```

You only need the ? for optional parameters